flips/README.md

```diff
@@ -0,0 +1,137 @@
+* be subject to discussion and a review meeting prior to acceptance
+
+## FLIP process
+
+Before submitting a FLIP, it is a good idea to discuss your aims with project
+contributors and maintainers and get early feedback. Use the [Flow community forum](https://forum.onflow.org/) or [Discord server](https://discord.gg/flow). After writing the FLIP draft, get feedback from these experts before submitting it.
+
+1. Recruit a sponsor from the maintainers of the project which your FLIP concerns.
+
+   Identify them in the FLIP, before posting the PR in step 2.
+   If no sponsor is found you may still post the FLIP, but if 
+   within a month of posting the PR there is still no sponsor,
+   it will be closed.
+
+2. Submit your FLIP as a pull request to [onflow/flow](https://github.com/onflow/flow). 
+
+   Name your FLIP file using the [template](./yyyymmdd-flip-template.md) `YYYYMMDD-descriptive-name.md`, where
+   YYYYMMDD is the date of submission, and ‘descriptive-name’ relates to the
+   title of your FLIP. For instance, if your FLIP is titled “Event Streaming API”,
+   you might use the filename `20180531-event-streaming-api.md`. If you have images
+   or other auxiliary files, create a directory of the form `YYYYMMDD-descriptive-name`
+   in which to store those files.
+
+   Include the header table and the contents of the **Objective** section 
+   in the comment of your pull request, using Markdown. For an example,
+   please see [this example FLIP](https://github.com/onflow/flow/pull/5). Include a mention of any of the GitHub handles of co-authors, reviewers, and sponsors.
+
+   At the top of the PR identify how long the comment period will be. This
+   should be a minimum of two weeks from posting the PR.
+
+3. Send a message to the #developers channel on [Discord](https://discord.gg/flow) with 
+   a brief description, and a link to the PR and a request for review.
+
+4. The sponsor may request a review committee meeting after sufficient discussion has 
+   taken place. This meeting will include the the FLIP author, core contributors and interested community members. If discussion is lively, wait until it has settled before going to review. The goal of the review meeting is to resolve minor issues; consensus should be reached on major issues beforehand.
+
+5. The meeting may approve the FLIP, reject it, or require changes before it
+   can be considered again. Approved FLIPs will be merged into `onflow/flow/flips`, and
+   rejected FLIPs will have their PRs closed.
+
+6. Implementations of a successful FLIP should reference it in their
+   documentation, and work with the sponsor to successfully land the code.
+
+While implementation code is not necessary to start the FLIP process, its
+existence in full or part may help the design discussion.
+
+If in any doubt about this process, feel free to ask on the
+developers mailing list or file an issue in [onflow/flow](https://github.com/onflow/flow/issues).
```

Comment:
What is the developers mailing list? Do we have something like that?

Reply:
Good catch. This was from the TensorFlow template -- I'll replace with Discord/forum since we don't have a mailing list.